### PR TITLE
Update js-sdk; Update iframe-bi-logger

### PR DESCRIPTION
**Pull request by bordi**
In order to met infra requirements, OOI apps should update:
- `js-sdk`
- `iframe-bi-logger` which is using in the Settings Panel app. 

I've updated those versions and tested on newly generated project (`create-yoshi-app:dev`)

Still need to test these versions with real app along with `tpa-settings` lib

### CODE REVIEW

File: packages/create-yoshi-app/package.json [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-    "find-cache-dir": "^2.0.0",
-    "minimist": "^1.2.0",
-    "mocha": "^5.2.0",
-    "mocha-env-reporter": "^3.0.0",
-    "mock-require": "^3.0.3",
-    "tempy": "^0.5.0",
-    "ts-node": "^8.4.1",
-    "type-fest": "^0.14.0"
-  }
-}
```
Comment (yavorsky) [inline packages/create-yoshi-app/package.json:69]: 🥺

**Comment by bordi** [comment]
@yavorsky another task, I guess,  OOI apps should not use `@wix/web-bi-logger` 
https://github.com/wix/yoshi/blob/master/packages/create-yoshi-app/templates/flow-editor/typescript/src/components/%7Beach%25flowData.components:name%25each%7D/Widget/Widget.tsx#L16

Instead, Viewer part should use logger provided by platform:
https://bo.wix.com/wix-docs/client/viewer-platform---ooi/articles/bi

File: packages/create-yoshi-app/package.json (hunk 1 above)
Comment (bordi) [reply packages/create-yoshi-app/package.json:69]: oops 😳

**Comment by bordi** [comment]
@yavorsky another task, I guess,  OOI apps should not use `@wix/web-bi-logger` 
https://github.com/wix/yoshi/blob/master/packages/create-yoshi-app/templates/flow-editor/typescript/src/components/%7Beach%25flowData.components:name%25each%7D/Widget/Widget.tsx#L16

Instead, Viewer part should use logger provided by platform:
https://bo.wix.com/wix-docs/client/viewer-platform---ooi/articles/bi

**Comment by bordi** [comment]
@yavorsky checked on Challenges app new sdk version - everything seems to be fine

**Comment by yavorsky** [comment]
Thanks!

**Pull request merged**